src/BaGet.UI/src/DisplayPackage/RepoDetails.tsx

```diff
@@ -0,0 +1,105 @@
+import { Icon } from 'office-ui-fabric-react/lib/Icon'
+import { Link } from 'office-ui-fabric-react';
+import * as React from 'react';
+
+import './RepoDetails.css'
+
+interface IRepoDetailsProps {
+  url: string;
+}
+
+interface IRepoDetailsState {
+  openIssues: number;
+  openPulls: number;
+  lastCommit: Date;
+  detailsFound: boolean;
+}
+
+class RepoDetails extends React.Component<IRepoDetailsProps, IRepoDetailsState> {
```

Comment:
Just curious, why did you create a new component instead of updating the [`SourceRepository`](https://github.com/loic-sharma/BaGet/blob/master/src/BaGet.UI/src/DisplayPackage/SourceRepository.tsx) component?

Reply:
Once I've finished this I might remove the heading tag and put it inside the SourceRepository component, this is a good point